utils/ssl_renewel_util/README.md

```diff
@@ -0,0 +1,82 @@
+# SSL Certificate Renewal Script for Aws
+Note: If you are not using Aws as your Dns provider then this script won't work. You'll have to manually renewel the certificates, In case if you are using any other Dns provider then check if certbot supports any dns plugin for your Dns provider and then just update the certbot command then the script should work fine.
```

Comment:
If you are not using Aws --> If you are not using Route 53 from Aws

Reply:
renewel --> renew

Reply:
updated